examples/trim.cf

```diff
@@ -0,0 +1,28 @@
+#+begin_src cfengine3
```

Comment:
Please make the changelog entry (commit title) past tense. For example:

```
Added string_trim() policy function
```

Reply:
:+1: